src/test/java/com/example/dog/controller/DogControllerRestAssuredMockMvcTest.java

```diff
@@ -31,10 +31,7 @@ public void setUp() {
 
     @Test
     public void possibleToGetExistingDog() {
-        Dog dog = new Dog("Tuzik", 24, 8,
```

Comment:
Why did you remove the date? You need to check it too..

Reply:
Because, first of all, I got `NullPointerException` If I passed `null` value. I should've checked this case since it was a problem to _format_ such ZonedDateTime when you `post` a dog (now it's fixed in code). Secondly, other test cases have a _normal_ date in post requests and I see no reason to duplicate this test with a dog that have a _normal_ date